doc/source/earthdata.md

```diff
@@ -36,29 +39,33 @@ a security issue for you, do not use this method.
 To store your credentials use the following code in an interactive Python session: 
 ```python
 from podpac.core.authentication import EarthDataSession
-eds = EarthDataSession()
-eds.update_login()
+
+EarthDataSession(username="<earthdata-username>", password="<earthdata-password>", store_credentials=True)
```

Comment:
It looks like this is out of date now. Did I miss something, or did you?

Reply:
whoops. forgot to update docs. will do this now